cloudpickle/cloudpickle.py

```diff
@@ -78,6 +78,14 @@
     string_types = (str,)
     PY3 = True
 
+if PY3:
+    from pickle import _getattribute
+else:
+    # compatibility function for python2 with the same signature as
+    # _getattribute
```

Comment:
Maybe extend the comment that under Python we do not need to implement the handling of dotted path as done in the Python 3 implementation of `_getattribute`.